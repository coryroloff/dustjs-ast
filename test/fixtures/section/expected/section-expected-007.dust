{#section key=value newKey=three.two.one /}