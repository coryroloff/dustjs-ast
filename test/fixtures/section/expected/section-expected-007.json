{
	"type": "Template",
	"body": [
		{
			"type": "Section",
			"body": [
				{
					"type": "Block",
					"body": [
						{
							"type": "Buffer",
							"text": "",
							"start": 8,
							"end": 8,
							"loc": {
								"start": {
									"line": 1,
									"column": 8
								},
								"end": {
									"line": 1,
									"column": 8
								}
							}
						}
					],
					"start": 8,
					"end": 8,
					"loc": {
						"start": {
							"line": 1,
							"column": 8
						},
						"end": {
							"line": 1,
							"column": 8
						}
					}
				}
			],
			"params": [],
			"kind": "+",
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "block",
						"start": 2,
						"end": 7,
						"loc": {
							"start": {
								"line": 1,
								"column": 2
							},
							"end": {
								"line": 1,
								"column": 7
							}
						}
					}
				],
				"localContext": false,
				"start": 2,
				"end": 7,
				"loc": {
					"start": {
						"line": 1,
						"column": 2
					},
					"end": {
						"line": 1,
						"column": 7
					}
				}
			},
			"start": 0,
			"end": 16,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"start": 0,
			"end": 1,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 1
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "+",
			"start": 1,
			"end": 2,
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "block",
			"start": 2,
			"end": 7,
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 7
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"start": 7,
			"end": 8,
			"loc": {
				"start": {
					"line": 1,
					"column": 7
				},
				"end": {
					"line": 1,
					"column": 8
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "{",
			"start": 8,
			"end": 9,
			"loc": {
				"start": {
					"line": 1,
					"column": 8
				},
				"end": {
					"line": 1,
					"column": 9
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "/",
			"start": 9,
			"end": 10,
			"loc": {
				"start": {
					"line": 1,
					"column": 9
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		},
		{
			"type": "Key",
			"value": "block",
			"start": 10,
			"end": 15,
			"loc": {
				"start": {
					"line": 1,
					"column": 10
				},
				"end": {
					"line": 1,
					"column": 15
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"start": 15,
			"end": 16,
			"loc": {
				"start": {
					"line": 1,
					"column": 15
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		}
	],
	"start": 0,
	"end": 16,
	"loc": {
		"start": {
			"line": 1,
			"column": 0
		},
		"end": {
			"line": 1,
			"column": 16
		}
	}
}